USDA_data.py

```diff
@@ -1,38 +1,31 @@
-import os
+import matplotlib.pyplot as plt
 import pandas as pd
+import os
+ 
+# Read input data into dataframe df
+df = pd.read_csv('USDA_data.csv', low_memory=False)
 
-# Data will be outputed to ./results directory
-os.makedirs('./results', exist_ok=True)
-
-# Read input data as DataFrame df
-df = pd.read_excel('USDA_data.xlsx')
-
-# Clean data by removing baselining and duplicates
-df = df[df['Usage'] != 'Baselining'].drop_duplicates()
-
-# Ignore application that skews visualizations
-df = df[df['Name'] != 'Adobe Reader and Acrobat Manager']
-
-# Indicate all columns where tag data may be located
-id_columns = [
-    'Asset - Custom Tags - Copy.2', 
-    'Asset - Custom Tags - Copy.3', 
-    'Asset - Custom Tags - Copy.4', 
-    'Asset - Custom Tags - Copy.5'
-    ]
-
-# Save all unique Mission Areas and Agency IDs into Set tags
+# Part 2: Save all unique Mission Areas and Agency IDs into set tags
 tags = set()
-for column in id_columns:
+# Indicate all columns where tag data may be located and save unique tags into a set 'tags'
+IDColumns = ['Asset - Custom Tags - Copy.2', 'Asset - Custom Tags - Copy.3', 'Asset - Custom Tags - Copy.4', 'Asset - Custom Tags - Copy.5']
+for column in IDColumns:
     tags.update(df[column].dropna().unique())
+# Make folders to store figures and data
+os.mkdir("./data")
+os.mkdir("./figures")
```

Comment:
What happens when these directories already exist? Might want to look at the `exist_ok` argument of `os.mkdir`.